src/openforms/emails/templates/emails/admin_digest.html

```diff
@@ -107,3 +107,42 @@ <h5>{% trans "Found invalid certificates" %}</h5>
     </ul>
 {% endif %}
 
+{% if invalid_registration_backends %}
+    <h5>{% trans "Registration backends problems" %}</h5>
+    <ul>
+        <ul>
+        {% for failed_backend in invalid_registration_backends %}
+            <li>
+                <p>
+                    {% blocktranslate with name=failed_backend.config_name message=failed_backend.exception_message trimmed %}
+                        The configuration for plugin '{{ name }}' is invalid.<br>
+                        ({{ message }})<br>
+                    {% endblocktranslate %}
+                </p>
+                <p>
+                    {% blocktranslate with form_name=failed_backend.form_name trimmed %}
+                        This affects form '{{ form_name }}'
+                    {% endblocktranslate %}
```

Comment:
this could affect more forms that use the same registration backend, right? In the case of a generic plugin check (without form-specific options). The easiest way to handle this would be a small language change:

```suggestion
                    {% blocktranslate with form_name=failed_backend.form_name trimmed %}
                        This affects form '{{ form_name }}', possibly other forms are affected too.
                    {% endblocktranslate %}
```